test/framework/streaming/stream_module.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var stream = require('stream');
+
+function hello(restbase, req) {
+    var body = new stream.PassThrough();
+    body.end('hello');
+    return {
+        status: 200,
+        headers: {
+            'content-type': 'text/html',
+        },
+        body: body
+    };
+}
+
+function buffer(restbase, req) {
+    var body = new stream.PassThrough();
+    body.end(new Buffer('hello'));
+    return {
+        status: 200,
+        headers: {
+            'content-type': 'text/html',
+        },
+        body: body
+    };
+}
+
+function chunks(restbase, req) {
+    var body = new stream.PassThrough();
+    for (var i = 0; i < 100; i++) {
+        body.write(i.toString());
+    }
+    body.end();
+    return {
+        status: 200,
+        headers: {
+            'content-type': 'text/html',
+        },
+        body: body
+    };
+}
+
+module.exports = function(options) {
+    return {
+        spec: {
+            paths: {
+                '/hello': {
+                    get: {
+                        operationId: 'hello'
+                    }
+                },
+                '/buffer': {
+                    get: {
+                        operationId: 'buffer'
+                    }
+                },
+                '/chunks': {
+                    get: {
+                        operationId: 'chunks'
+                    }
+                }
+            }
+        },
+        operations: {
+            hello: hello,
+            buffer: buffer,
+            chunks: chunks,
+        }
+    };
+};
```

Comment:
Could be nice to add an async test, that calls `body.end()` from a timeout callback, to verify that async writing also work.